scripts/liquidation_bot/liquidateUnderwaterBorrowers.ts

```diff
@@ -19,49 +20,45 @@ export interface Asset {
   scale: bigint;
 }
 
-const daiPool = {
-  tokenAddress: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
-  poolFee: 100
-};
-
-async function sendFlashbotsPrivateTransaction(
-  txn: PopulatedTransaction,
-  flashbotsProvider: FlashbotsBundleProvider
-) {
-  const privateTx = {
-    transaction: txn,
-    signer: flashbotsProvider.getSigner(),
-  };
-  await flashbotsProvider.sendPrivateTransaction(privateTx);
-}
-
-async function sendTxn(
-  txn: PopulatedTransaction,
-  signerWithFlashbots: SignerWithFlashbots
-) {
-  if (signerWithFlashbots.flashbotsProvider) {
-    googleCloudLog(LogSeverity.INFO, 'Sending a private txn');
-    await sendFlashbotsPrivateTransaction(txn, signerWithFlashbots.flashbotsProvider);
-  } else {
-    googleCloudLog(LogSeverity.INFO, 'Sending a public txn');
-    await signerWithFlashbots.signer.sendTransaction(txn);
+const flashLoanPools = {
+  'mainnet': {
+    // DAI pool
+    tokenAddress: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
+    poolFee: 100
+  },
+  'goerli': {
+    // WETH pool
```

Comment:
does it matter that we use `WETH` both as the pair token to initiate the flash loan and also as an intermediary step in swapping?

Reply:
once we merge #623 we can also set a conservative `maxCollateralToPurchase` value if that will help use a different pool